server/src/completions/reserved-variables.ts

```diff
@@ -0,0 +1,136 @@
+export const RESERVED_VARIABLES = [
```

Comment:
I'm not sure I understand what are "reserved variables" in the context of bitbake. They can all freely be modified, just like regular variables.

Reply:
In my understanding, `reserved` means this piece of code is unique and its usage is not to be messed with others. In the case of bitbake variables, for example, `BBPATH` is always used for path to the .bbclass and .conf and bitbake engine will never look for `BBPATH_1` for the path right? 

But if you are saying it means `can't be modified` whatsoever in general, then I could just rename it as `BITBAKE_VARIABLES` to avoid confusion.

Reply:
Renamed